shadow-cljs.edn

```diff
@@ -106,4 +106,13 @@
            :output-to "target/mocks/mocks.js"
            :output-dir "target/mocks"
            :compiler-options {:optimizations :simple
-                              :source-map false}}}}
+                              :source-map false}}
+          :component-test {:target :npm-module
+                           :entries [quo2.components.selectors.--tests--.selectors-component-spec]
```

Comment:
Why do have this namespace hardcoded here?

From the `:entries` doc:

> The namespaces that serve as the root nodes of the dependency graph for the output code of this module.

This namespace doesn't look like a root node. Perhaps we could use something like the following and create an entry point namespace. This has the added benefit of being REPL friendly instead of relying on the shadow-cljs shenanigans.

```clojure
:entries [test.components.core]
```

Reply:
yep we could do that